packages/components/stories/config/i18n.js

```diff
@@ -0,0 +1,21 @@
+import i18n from 'i18next';
+
+i18n
+	.init({
+		// locales load path. lng = language, ns = namespace
+		// backend: {
+		// 	loadPath: '/locales/{{lng}}/{{ns}}.json',
+		// },
+
+		// have a common namespace used around the full app
+		// ns: ['common'],
+		// defaultNS: 'common',
+
+		// Fallback language
+		// fallbackLng: 'en',
+		debug: false,
+		wait: true, // globally set to wait for loaded translations in translate hoc
+	});
+
+
+export default i18n;
```

Comment:
Maybe add a comment to indicate that in the future, it is in this file where we will get JSON translation file with https://github.com/i18next/i18next-xhr-backend